Bio/SearchIO/ExonerateIO/_base.py

```diff
@@ -28,40 +29,80 @@ def _set_frame(frag):
     frag.query_frame = (frag.query_start % 3 + 1) * frag.query_strand
 
 
-def _make_triplets(seq):
-    """Splits a string into a list containing triplets of the original
-    string."""
-    return [seq[3 * i:3 * (i + 1)] for i in range(len(seq) // 3)]
+def _make_triplets(seq, phase=0):
+    """Given a sequence of three-letter amino acid codes and its phase,
```

Comment:
This ought to have a one line description for PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings.

e.g.

``` python
def _make_triplets(seq, phase=0):
    """Split sequence into partial, complete, and partial codons (PRIVATE).

    Given a sequence of three-letter amino acid codes and its phase,
    return the letters discarded to reach the first full triplet, the
    triplets, and any remaining non-triplets.
    """